pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -278,24 +285,44 @@ internal fun Pager(
     }
 }
 
-private object ConsumeFlingNestedScrollConnection : NestedScrollConnection {
+private class ConsumeFlingNestedScrollConnection(
+    private val consumeHorizontal: Boolean,
+    private val consumeVertical: Boolean,
```

Comment:
just curious: why is `isVertical: Boolean` not enough? :)

Reply:
It would have been. I just wanted to keep it more generalised, so people can copy-pasta it if they need, etc.